lib/index.js

```diff
@@ -4,6 +4,10 @@ var stringify = require('./stringify');
 var parse = require('./parse');
 
 module.exports = {
+    formats: {
```

Comment:
Hm, include the file for two strings gives uncleared and unjustified overhead. And in the same time - it replaces the only one occurrence in stringify.js : input parameter validation.
And mix constants and formatters to the one file will be out of KISS